h/static/scripts/test/annotation-ui-test.js

```diff
@@ -49,6 +49,21 @@ describe('annotationUI', function () {
       annotationUI.removeAnnotations([annot]);
       assert.deepEqual(annotationUI.getState().annotations, []);
     });
+
+    it('matches annotations to remove by ID', function () {
+      var annot = annotationFixtures.defaultAnnotation();
+      annotationUI.addAnnotations([annot]);
+      annotationUI.removeAnnotations([{id: annot.id}]);
+      assert.deepEqual(annotationUI.getState().annotations, []);
```

Comment:
I know it's a bit pedantic, but a minimal passing implementation for this test would be one that simply removed all annotations, regardless of id, tag, or anything else. Perhaps it would be wise to add a couple more annotations before performing the remove just so it's a bit clearer that it's doing the right thing?